### PR TITLE
Adds blog post for SCALE '17 conference

### PR DESCRIPTION
**Pull request by bmbouter**

### CODE REVIEW

File: _posts/2017-03-01-pulp-at-scale-2017.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+title: Pulp at SCALE '17
+author: Brian Bouterse
+---
+Pulp is participating in the Southern CAlifornia Linux Expo
+([SCALE](http://www.socallinuxexpo.org/scale/15x)) from March 2-5, 2017 in Pasadena, California.
+Michael Hrivnak will be giving a talk titled *[Deploying a Multi-Service App in Kubernetes](
+http://www.socallinuxexpo.org/scale/15x/presentations/deploying-multi-service-app-kubernetes)* which
+deploys Pulp itself using containers and Kubernetes. Brian Bouterse will be giving a talk titled
```
Comment (tehsmyers) [inline _posts/2017-03-01-pulp-at-scale-2017.md]: It seems weird to talk about yourself in the third person in an authored message.

File: _posts/2017-03-01-pulp-at-scale-2017.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+*[Debugging Hung Python Processes with GDB](http://www.socallinuxexpo.org/scale/15x/presentations/debugging-hung-python-processes-gdb)*
+which was well received at FOSDEM.
+
+These conferences are a great place for Pulp users and developers to meet-up. We are looking at
```
Comment (tehsmyers) [inline _posts/2017-03-01-pulp-at-scale-2017.md]: I think s/are looking at/will be posting to/ might be good, but I'm not sure.

**Review by tehsmyers — APPROVED** [review]

File: _posts/2017-03-01-pulp-at-scale-2017.md (hunk 1 above)
Comment (bmbouter) [reply _posts/2017-03-01-pulp-at-scale-2017.md]: good idea. fixed in next push

File: _posts/2017-03-01-pulp-at-scale-2017.md (hunk 2 above)
Comment (bmbouter) [reply _posts/2017-03-01-pulp-at-scale-2017.md]: I was hoping you would suggest some thing here. Fixed in next version.

**Pull request merged**